### PR TITLE
Responsive cards

### PR DESCRIPTION
**Pull request by ramshorst**

### CODE REVIEW

File: less/base/_cards.less [hunk 1]
```diff
@@ -1,9 +1,37 @@
...
+  display: inline-block;
+  height: 260px;
+  width: 330px;
+  margin: @spacing-xx-sm;
   padding: @spacing-x-lg @spacing-x-sm;
   border: 2px solid @upf-gray;
   border-radius: @default-radius;
-  height: 260px;
-  text-align: center
+  display: inline-block;
```
Comment (tgallice) [inline less/base/_cards.less]: Already defined L.26

Comment (ramshorst) [reply less/base/_cards.less]: Thanks :)

**Comment by AlexisMontagne** [comment]
Sounds good to me, @phndiaye what do you think of it?

**Comment by phndiaye** [comment]
LGTM 👍

**Pull request merged**